appserver/jdbc/jdbc-config/src/main/java/org/glassfish/jdbc/config/validators/JdbcConnectionPoolValidator.java

```diff
@@ -71,17 +72,58 @@ public boolean isValid(final ResourcePool pool,
 
         if(poolFaults == ConnectionPoolErrorMessages.MAX_STEADY_INVALID) {
             if(pool instanceof JdbcConnectionPool) {
+                
+                Properties systemProps = System.getProperties();
+                
                 JdbcConnectionPool jdbcPool = (JdbcConnectionPool) pool;
                 String maxPoolSize = jdbcPool.getMaxPoolSize();
                 String steadyPoolSize = jdbcPool.getSteadyPoolSize();
```

Comment:
Whats an example string for steadyPoolSize? For context

Reply:
It would be an integer value representing the minimum size of the connection pool (default value is "8")

Reply:
I meant the raw value, sorry, as I see you do some string manipulation, 'twas trying to figure out what you're trying to extract in that case.

Reply:
So in the event that a system variable has been passed in, ${MAX_POOL_SIZE}, for example - that will cause an error when running the Integer.parseInt method (for obvious reasons) so in that case the method will retrieve the supposedly integer value stored in that system variable rather than trying to parse the variable name string instead